nestkernel/connection_manager.cpp

```diff
@@ -395,6 +395,8 @@ nest::ConnectionManager::connect( NodeCollectionPTR sources,
   ConnBuilder* cb = connbuilder_factories_.at( rule_id )->create( sources, targets, conn_spec, syn_specs );
   assert( cb != 0 );
 
+  set_have_connections_changed( kernel().vp_manager.get_thread_id() );
```

Comment:
This should move down to right before the `cb->connect()` call to make the code more understandable. Probably it would be good to add a comment like "Set flag before calling cb->connect() in case exception is thrown after some connections have been created."